projects/Mallard/src/helpers/storage.ts

```diff
@@ -76,6 +78,17 @@ const lightboxSettingCache = createAsyncCache<boolean>('lightbox-enabled')
 
 const enableEditionMenuCache = createAsyncCache<boolean>('edition-menu-enabled')
 
+const selectedEditionCache = createAsyncCache<RegionalEdition | SpecialEdition>(
+    'selectedEdition',
+)
+
+const defaultEditionCache = createAsyncCache<RegionalEdition>('defaultEdition')
```

Comment:
Default Edition can only be `RegionalEdition` due to the expiry of `SpecialEdition`